manifests/core/type/hub/storage/generic.yaml

```diff
@@ -0,0 +1,111 @@
+ocfVersion: 0.0.1
+revision: 0.1.0
+kind: Type
+metadata:
+  name: generic
+  prefix: cap.core.type.hub.storage
+  displayName: Generic storage backend schema
+  description: Defines schema which is used for validation of backend storages.
+  documentationURL: https://capact.io/docs
+  supportURL: https://capact.io
+  maintainers:
+    - email: team-dev@capact.io
+      name: Capact Dev Team
+      url: https://capact.io
+spec:
+  jsonSchema:
+    value: |-
+      {
```

Comment:
Looks pretty good 👌  However, we should tweak it a bit as I discovered one bug.

- enforce exact number of items in the `required` property
- instead of `max` and `minProperties`, we can disable adding `additionalProperties` and also require exact properties names with `required`
   - This solves a bug, where a following schema is considered as valid:

```json
{
  "$schema": "http://json-schema.org/draft-07/schema",
  "type": "object",
  "title": "The root schema",
  "required": [
    "url",
    "acceptValue",
    "contextSchema"
  ],
  "properties": {
    "foo": {},
      "contextSchema": {
            "$id": "#/properties/contextSchema",
            "type": "null",
            "const": null
          },
    "acceptValue": {
      "$id": "#/properties/acceptValue",
      "type": "boolean",
      "const": false
    }
  },
  "additionalProperties": false
}
``` 

Tweaked schema:
```json
{
  "$schema": "http://json-schema.org/draft-07/schema",
  "type": "object",
  "title": "The generic storage backend schema",
  "required": [
    "required",
    "properties",
    "additionalProperties"
  ],
  "properties": {
    "required": {
      "$id": "#/properties/required",
      "type": "array",
      "maxItems": 3,
      "allOf": [
        {
          "contains": {
            "const": "url"
          }
        },
        {
          "contains": {
            "const": "acceptValue"
          }
        },
        {
          "contains": {
            "const": "contextSchema"
          }
        }
      ]
    },
    "properties": {
      "$id": "#/properties/properties",
      "type": "object",
      "required": [
        "contextSchema",
        "url",
        "acceptValue"
      ],
      "properties": {
        "url": {
          "type": "object",
          "properties": {
            "type": {
              "const": "string"
            }
          },
          "required": [
            "type"
          ]
        },
        "contextSchema": {
          "type": "object",
          "oneOf": [
            {
              "properties": {
                "const": {
                  "type": "object"
                }
              },
              "required": [
                "const"
              ]
            },
            {
              "properties": {
                "type": {
                  "const": "null"
                }
              },
              "required": [
                "type",
                "const"
              ]
            }
          ]
        },
        "acceptValue": {
          "type": "object",
          "properties": {
            "type": {
              "const": "boolean"
            }
          },
          "required": [
            "type",
            "const"
          ]
        }
      },
      "additionalProperties": false
    },
    "additionalProperties": {
      "const": false
    }
  }
}
```